Assets/MixedRealityToolkit-SDK/Features/Input/Scripts/MixedRealityInputManager.cs

```diff
@@ -19,7 +17,7 @@
 using UnityEngine;
 using UnityEngine.EventSystems;
 
-namespace Microsoft.MixedReality.Toolkit.InputSystem
+namespace Microsoft.MixedReality.Toolkit.SDK.Input
```

Comment:
Not sure the Input manager should move to the SDK, as it is one of our core features.
Granted you currently have dependencies on Focus / Gaze, which moved to core.  but this should mean refactoring the implementation rather than moving it.

Reply:
I think it makes sense after our conversation during the ship room meeting.

Reply:
It's our specific implementation.

Reply:
OK, I'll let it go for now.  But that would ring true for ANY component we build?  Making the Core just definitions, interfaces and the MRManager. (meaning out of the box, the core does nothing but start an empty scene)

Reply:
>  (meaning out of the box, the core does nothing but start an empty scene)

I'm with @SimonDarksideJ on this one... I believe we will want to move our implementations back into the core.

Reply:
There are very likely going to be devs that want our base implementations and *nothing else* from the SDK packages.

Reply:
>There are very likely going to be devs that want our base implementations and nothing else from the SDK packages.

I can see this being the case for a few, but at that point they might as well be writing their own implementation and can just cherry pick what they want via the SDK.

Reply:
except that the sdk packages will likely contain much more than they want

Reply:
let's keep the default managers (input, boundary, etc) in the core and everything else in an sdk package

Reply:
So then should we also move the default profiles as well?

Reply:
that sounds like a great idea.

Reply:
I say our "default profiles" are a different thing entirely, as they are "quick start" things.

They are not crucial to someone downloading the Core project to get running as they can quickly create their own.  

The profiles in the SDK are aimed at making the SDK easy to pick up and get running only.

Reply:
@SimonDarksideJ, that makes complete sense.

Reply:
I'm inclined to keep the manager implenetations and the profiles together.

I don't see them as mutually exclusive things.

Reply:
@StephenHodgson, as does your comment...

I am leaning more and more to moving the default configuration files to _Core to live beside the default managers.

Reply:
I'm losing my hair.  Our "default" configuration profiles are merely examples. Even in the SDk, users are going to customise them and create their own.  Our "defaults" are the settings we configure in the scripable objects in Core,  so there isn't any need for the .asset file to live in core.

So scriptable config definitions are in core (as they are now), the "example" profiles should live in the SDK as defaults.  One could even argue they don't even need to exist at all, but they are good defaults.

Reply:
How about we table this PR until after Alpha and then spend the appropriate amount of design / discussion time to ensure we get it right for Beta.

Reply:
I think the PR as a whole is fine.  We can always shuffle again later before we hit the Alpha button
Trying to focus on the visualizer and I'm also adding the Rift config now (as that's what is plugged in), also sorted out multi-select controls now for OpenVR.

Going to need to do another "controllers" refactor to simplify the code, later.

Reply:
That makes sense. Let's get this in and shuffle next week (or right after Alpha). We are a monolithic package for this release anyhow :)